doc/theory/sh_basis.rst

```diff
@@ -57,28 +68,42 @@ literature for the computation of the ODF. DIPY implements two of these in the
 
     Y_i(\theta, \phi) =
      \begin{cases}
-     \Im(Y_l^{|m|}(\theta, \phi)) & -l \leq m < 0, \\
+     \Im(Y_l^{m}(\theta, \phi)) & -l \leq m < 0, \\
      Y_l^0(\theta, \phi) & m = 0, \\
      \Re(Y_{l}^m(\theta, \phi)) & 0 < m \leq l
      \end{cases}
 
 In both cases, $\Re$ denotes the real part of the spherical harmonic basis, and
-$\Im$ denotes the imaginary part.
+$\Im$ denotes the imaginary part. The SH bases are both orthogonal and real. Moreover,
+the `descoteaux07` basis is orthonormal. 
```

Comment:
Double backticks, right?

Reply:
Looking at this [page](https://dipy.org/documentation/1.2.0./theory/sh_basis/) from the documentation for DIPY 1.2, `$\Im$` and `$\Re$` appear to be formatted the right way. Should I add  the double backticks around the `$` symbol?

Reply:
True, my bad ! No, the rendered doc looks right an consistent with the LaTeX symbols used in the `cases`, so no changes are needed.